Tests/Runner/DataLoaderTest.php

```diff
@@ -481,4 +487,84 @@ public function testBranchMappingEnabled()
         self::assertInstanceOf(Result::class, $storageState->getInputTableResult());
         self::assertInstanceOf(InputFileStateList::class, $storageState->getInputFileStateList());
     }
+
+    public function testFixedTypeTableCreate()
```

Comment:
also here, why is it "Fixed" ?

Reply:
I dunno man 😄 .  I'll change it to just TypedTable, so `testTypedTableCreate` ok?